nav.yml

```diff
@@ -1,1205 +1,1206 @@
 nav:
   - Using n8n:
```

Comment:
Could you reset this file back and only add the new entries? We're in the process of doing a big reorg of the docs navigation on another branch and I think only having the targetted changes would help us when we integrate those.

Reply:
I am not sure I understand, I *think* I only added a link to the new page...

Reply:
If you look at the diff, it looks like maybe your editor made some formatting changes (or maybe need to merge in `main`)? I'm not sure, but just going off of the diff.

Reply:
Oh you are right, I will fix asap.

Reply:
Should be good now, apologies for the noisy formatting.

Reply:
No problem, thanks for the fix!